README.md

```diff
@@ -59,7 +59,6 @@ Make sure you have `virtualenv` installed.
 - Transform configuration dictionary into a parameter dictionary using the registered configuration readers
 - Update the parameter dictionary with any overrides from the registered cli argument readers
 - Transform the parameter dictionary into a list of formatted parameters
-- Append unknown arguments if the `--unknown_arguments` flag is set
```

Comment:
I think you'll need to delete the `unknown_arguments` from the `UtilCliparser` where it's registered.